settings.gradle

```diff
@@ -8,11 +8,11 @@ pluginManagement {
   }
 
   plugins {
-    def kotlinVersion = settings.hasProperty("kotlinDev") ? "1.4.0-rc" : "1.3.70"
+    def kotlinVersion = settings.hasProperty("kotlinDev") ? '1.4.0-rc' : '1.4.0'
```

Comment:
Also here.

Reply:
Also had this idea (see first commit), but found this in readme file

> Additionally, project publishes snapshots build against latest kotlin development version. To use them, change version
> of ktlint to `<latest-version>-kotlin-dev-SNAPSHOT`.
I guess, this is added to test ktlint against any other Kotlin dev version.